File: tests/src/main/java/com/github/klee0kai/test/cache/di/ann/StoneStrongScope.java
package com.github.klee0kai.test.cache.di.ann;

import javax.inject.Scope;
import java.lang.annotation.Retention;
import java.lang.annotation.Target;

import static java.lang.annotation.ElementType.METHOD;
import static java.lang.annotation.RetentionPolicy.RUNTIME;

@Scope
@Retention(RUNTIME)
@Target(METHOD)
public @interface StoneStrongScope {
}

File: test_kotlin/src/main/kotlin/com/github/klee0kai/test/cache/di/ann/StoneStrongScope.kt
package com.github.klee0kai.test.cache.di.ann

import javax.inject.Scope

@Scope
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
annotation class StoneStrongScope 